ibm/service/kubernetes/resource_ibm_container_cluster.go

```diff
@@ -781,7 +723,38 @@ func resourceIBMContainerClusterCreate(d *schema.ResourceData, meta interface{})
 
 	d.Set("force_delete_storage", d.Get("force_delete_storage").(bool))
 
-	return resourceIBMContainerClusterUpdate(d, meta)
```

Comment:
Yes, this is the point of this change, after cluster creation the labels etc set up in one go. but these attributes are actually belonging to workerpools, and only worked on the default workerpool. It still make the changes in one go. If update is required, it still required another run of terraform. 

However updates to workerpool (and now that includes the default workerpool too) now expected to be done through its resource and not through cluster resource. Now the default workerpool works the same way as other workerpools. And to have it ready there is the import_on_create field for it, so no manual terrafrom import is required. 

Currently there default workerpool can also be modified by workerpool resource and if values mismatching between the cluster and the workerpool resource, it can cause problem, this possibility we are aiming to elinimate.

Reply:
I added back the update call, as the workerpool related part that should only happen once has been moved to create. Thus update should be safe to call and this gives back the possibility add cluster related changes too, and only make it once, instead of making it for create and update too. And should not affect the workerpools anymore.

Reply:
Can we have same changes made for vpc-cluster also?

Reply:
i did